Sources/Xcodeproj/pbxproj().swift

```diff
@@ -146,7 +146,8 @@ func xcodeProject(
     projectSettings.common.COMBINE_HIDPI_IMAGES = "YES"
 
     // Defined for regular `swift build` instantiations, so also should be defined here.
-    projectSettings.common.SWIFT_ACTIVE_COMPILATION_CONDITIONS += ["SWIFT_PACKAGE"]
+    projectSettings.common.SWIFT_ACTIVE_COMPILATION_CONDITIONS += ["$(inherited)", "SWIFT_PACKAGE"]
```

Comment:
There's code to always add `"$(inherited)"` to the projectSettings.common.SWIFT_ACTIVE_COMPILATION_CONDITIONS elsewhere in the file - maybe this isn't necessary?

Reply:
That's for target level